src/components/app/MainContent.vue

```diff
@@ -36,48 +68,56 @@ const statusToColor = {
 
 export default defineComponent({
   components: {
-    LogViewer, ConfigEditor
+    ConfigEditor,
+    LogViewer
   },
   setup() {
+    const config = store.getters.Config;
     const scrapingStatus = ref('NOT_STARTED' as keyof typeof statusToColor);
+    const generalError = ref<string>('');
     const inProgress = computed(() => scrapingStatus.value === 'IN_PROGRESS');
     const btnColor = computed(() => statusToColor[scrapingStatus.value]);
-    const entries = ref([] as LogEntry[]);
+    const showAccountCards = computed(() => scrapingStatus.value !== 'NOT_STARTED');
 
-    const eventPublisher = LogsEventEmitter((entry) => entries.value.push(entry));
+    const accountsState = reactive(new AccountsState(config.getActiveImporters, config.getActiveExporters));
```

Comment:
I think because only the **initial value** is comes from the **store**, when the store changed, the `accountsState` not updating.

For example, try to run once (to load the component) then add another Importer or enable Exporter, and it will not show.

Reply:
Thats true. Solved this by creating a new AccountsState every time we start scraping. This actually makes sense because this basically contains the status of the process so every time we run again the status starts from scratch.